### PR TITLE
LPD-24499 Improve restricted content message

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-24499

### CODE REVIEW

File: modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-impl/src/main/java/com/liferay/fragment/entry/processor/internal/util/FragmentEntryProcessorHelperImpl.java [hunk 1]
```diff
@@ -801,6 +793,43 @@ else if (isMappedDisplayPage(editableValueJSONObject)) {
 		return new InfoItemMappedField(fieldName, infoItemReference, object);
 	}
 
+	private String _getRestrictedContentMessage(
+		FragmentEntryProcessorContext fragmentEntryProcessorContext) {
+
+		StringBundler sb = new StringBundler(7);
+
+		sb.append("<div class=\"alert alert-secondary ");
```
Comment (ealonso) [inline modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-impl/src/main/java/com/liferay/fragment/entry/processor/internal/util/FragmentEntryProcessorHelperImpl.java:801]: @lfbesada there is an alert taglib, Could be possible to use it?

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
@ealonso Thanks a lot for the tip! Much better in this way :heart: Could you please take a look? Thank you!

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**